### PR TITLE
Blog on oVirt game in GetBadges

### PR DESCRIPTION
**Pull request by eedri**
Signed-off-by: Eyal Edri <eedri@redhat.com>

Initial draft for oVirt gamification blog.

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

**Comment by sandrobonazzola** [comment]
:-1: still GetBadgets instead of GetBadges. @getbadges CEO took the time to review the blog post and deserve to be addressed

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+To put it in other words, it is turning day-to-day tasks, the kind you might do at home or work, into a game which you can
+earn points, badgets and compete with other people that are doing the same things.  
+
+## oVirt & Gamification
+
+You probably didn't know, but this isn't the first time oVirt gamification is being used. A few years ago there was an initiative to use oVirt UI plugins system to add Gamification to the project, there was even a "space invaders" game [written and available to play inside oVirt](https://www.ovirt.org/community/activities/gamification/)!
+
+### So What is New?
+
+The oVirt infra team recently reached out to 'GetBadges,' a company which provides 'Gamifcation as a Service'.
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: typo -> Gamification

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Luckly for us, open source projects gets to have a free game! So oVirt was rewarded with its own [oVirt Open Source Game](https://ovirt-ovirt-engine.getbadges.io/activity).
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: gets->get

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 3]
```diff
@@ -1,53 +1,52 @@
...
-* [FAQ Part II](https://getbadges.io/blog/frequently-asked-questions-pt-2)<br>
-* [GetBadges Blog](https://getbadges.io/blog/)<br>
-* [Understanding Badges](https://getbadges.io/blog/understanding-badges-in-our-it-gamification-platform)<br>
-* [How to motivate developers](http://getbadges.io/blog/how-to-motivate-developers)<br>
+### oVirt 2016 Game Stats
 
+So you might ask yourself how are you doing so far in the game. Checkout the 2016 statistics shown in the infographic below. 
 
-## References
+![oVirt Game 2016 Statistics](/images/oVirt-game-stats.png)
```
Comment (eedri) [inline source/blog/2017-02-25-oVirt-gamification.md]: @bproffitt  is it possible to show this image as part of the blog instead of link to external image?

**Comment by eedri** [comment]
@bproffitt thanks for the updated content and fixes!
Just one thing before we can merge this, I'd like to know if its possible to embed the 2016 stats as part of the blog rather than external link

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+To put it in other words, it is turning day-to-day tasks, the kind you might do at home or work, into a game which you can
+earn points, badges and compete with other people that are doing the same things.  
+
+## oVirt & Gamification
+
+You probably didn't know, but this isn't the first time oVirt gamification is being used. A few years ago there was an initiative to use oVirt UI plugins system to add Gamification to the project, there was even a "space invaders" game [written and available to play inside oVirt](https://www.ovirt.org/community/activities/gamification/)!
+
+### So What is New?
+
+The oVirt infra team recently reached out to 'GetBadges', a company which provides 'Gamifcation as a Service'.
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: Typo : should be Gamification

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+Luckly for us, open source projects gets to have a free game! So oVirt was rewarded with its own [oVirt Open Source Game](https://ovirt-ovirt-engine.getbadges.io/activity).
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: Luckly -> Luckily
gets -> get

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+The game works automagically every time you contribute to the project. Current integrations are only active on specific projects like 'ovirt-engine' and 'vdsm'. If you wish to add your project to the game, please reach out to oVirt infra team and ask to add it. Current activation channels are: 
+
+    * GitHub  - any commit merged rewards points 
+    * Jenkins - any successful build rewards points and failed builds deduct points 
+    * Jira    - any new ticket opened or closed in jira.ovirt.org rewards points
+
+Other than points, you are rewarded with cool badges every time you acomplish an achievement, like triggering a first successful build. There are more cool features like slaying monsters :), you can read more detailed info on the GetBadges links below.
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: acomplish -> accomplish

File: source/blog/2017-02-25-oVirt-gamification.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+### oVirt 2016 Game Stats
+
+So you might ask yourself how are you doing so far in the game. Checkout the 2016 statistics shown in the infographic below. 
+
+![oVirt Game 2016 Statistics](/images/oVirt-game-stats.png)
+
+## More Info on GetBadges and the Game Elements
+
+You can find more info and details on the GetBadges game on the thier blog site.
```
Comment (rollandf) [inline source/blog/2017-02-25-oVirt-gamification.md]: thier -> their

**Comment by eedri** [comment]
@rollandf Thanks for all the reviews!

**Comment by bproffitt** [comment]
@eedri The way I have it set up it should be an image in the blog, so it should be the way you want it.

**Comment by eedri** [comment]
@bproffitt Great! so ACK from me to merge.

**Comment by bproffitt** [comment]
Merged #822

**Pull request merged**